src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -92,8 +92,15 @@ export class HttpResourceFetcher implements ResourceFetcher {
             return contents
         } catch (err) {
             const error = err as CancelError | { message?: string; code?: number }
-            this.logger.verbose(`Error downloading ${this.logText()}: %s`, error.message ?? error.code)
+            if (error instanceof CancelError) {
+                this.logger.verbose(
+                    `Error downloading ${this.logText()}: %s`,
+                    error.message ?? error.code ?? error.response?.statusMessage ?? error.response?.statusCode
```

Comment:
why is `instanceof CancelError` needed? is TS complaining about this line? Could avoid duplication by removing the `if (instanceof CancelError)` check?

Reply:
Yeah this is actually a subtle problem caused by `getResponseFromGetRequest`. It strips out everything but `code` (this was legacy). I'm guessing this was done because we were logging the entire error at one point? I'd suggest removing that catch block entirely. Then check for a `RequestError` here and log accordingly.

Reply:
The instanceof was just because typescript would throw an error on `error.response.statusMessage` because only CancelError in the type union had `response`.

I've just pushed a new change that does as Jaden suggested and the instanceof is no longer even needed